components/dashboard/src/projects/ConfigureProject.tsx

```diff
@@ -97,12 +95,17 @@ export default function () {
       const guessedConfigStringPromise = getGitpodService().server.guessProjectConfiguration(project.id);
       const repoConfigString = await getGitpodService().server.fetchProjectRepositoryConfiguration(project.id);
       if (repoConfigString) {
-        // TODO(janx): Link to .gitpod.yml directly instead of just the cloneUrl.
         setIsDetecting(false);
         setEditorMessage(<EditorMessage type="warning" heading="Configuration already exists in git." message="Run a prebuild or open a new workspace to edit project configuration."/>);
```

Comment:
**issue**: This is probably out of the scope of these changes but noticed that it takes a while to redirect to the configuration page once you select a team. Is this expected or needed? Cc @AlexTugarev

Reply:
True, @AlexTugarev noticed a UI freeze, and I did too today. Maybe it's indeed MonacoEditor? (It's loaded lazily in the Configurator, in order not to make the entire dashboard slower)